# ChangeLog for dev-python/oauth
# Copyright 1999-2013 Gentoo Foundation; Distributed under the GPL v2
# $Header: /var/cvsroot/gentoo-x86/dev-python/oauth/ChangeLog,v 1.5 2013/06/29 12:25:59 idella4 Exp $

*oauth-1.0.1-r1 (29 Jun 2013)

  29 Jun 2013; Ian Delaney <idella4@gentoo.org> +oauth-1.0.1-r1.ebuild,
  -oauth-1.0.1.ebuild:
  revbump; remove old

  21 Aug 2011; Pawel Hajdan jr <phajdan.jr@gentoo.org> oauth-1.0.1.ebuild:
  x86 stable wrt bug #378589

  14 Aug 2011; Markos Chandras <hwoarang@gentoo.org> oauth-1.0.1.ebuild:
  Stable on amd64 wrt bug #378589

  14 Jul 2010; Arfrever Frehtes Taifersar Arahesis <arfrever@gentoo.org>
  oauth-1.0.1.ebuild:
  Set SUPPORT_PYTHON_ABIS.

*oauth-1.0.1 (02 Mar 2010)

  02 Mar 2010; Jesus Rivero <neurogeek@gentoo.org> +oauth-1.0.1.ebuild,
  +metadata.xml:
  Initial commit. Closes bug #290159. Thanks to Ravi for the report
